examples/python/trigger-pagerduty-incident/stack.yml

```diff
@@ -0,0 +1,15 @@
+provider:
+  name: openfaas
+  gateway: https://127.0.0.1:31112
+functions:
+  pdinvoke-fn:
+    lang: python3
+    handler: ./handler
+    image: pkbu/veba-pagerduty-invoke:latest
+    environment:
+      write_debug: true
+      read_debug: true
+    secrets:
+      - pdconfig
+    annotations:
+      topic: VmPoweredOnEvent, VmPoweredOffEvent
```

Comment:
Please remove the whitespace after the ","

Reply:
curious, is this change requested as a yaml/k8s best practice?

Reply:
tbh, not sure how YAML interprets spaces in strings - perhaps it just passes them through as is. and in this case it's up to the library to interpret. would have to look up how the openfaas controller SDK handles whitespaces, but I think it's always safer to not assume correct whitespace handling and thus remove the space(s)

Reply:
done